packages/manager/modules/pci/src/projects/project/storages/databases/database/namespaces/add/add.html

```diff
@@ -0,0 +1,294 @@
+<oui-back-button
+    data-previous-page="{{:: 'pci_databases_namespaces_add_back' | translate }}"
+    data-on-click="$ctrl.trackAndGoBack()"
+></oui-back-button>
+
+<h2 data-translate="pci_databases_namespaces_add_title"></h2>
+
+<form name="$ctrl.addNamespaceForm" ng-submit="$ctrl.add()">
+    <div class="row">
+        <div class="col">
+            <oui-field>
+                <label class="oui-field__label">
+                    <span
+                        data-translate="{{:: 'pci_databases_namespaces_add_name_label' | translate }}"
+                    ></span>
+                    <small class="text-danger">*</small>
+                </label>
+                <input
+                    class="oui-input"
+                    type="text"
+                    id="namespace_name"
+                    name="namespace_name"
+                    ng-model="$ctrl.model.name"
+                    ng-pattern="$ctrl.FORM_RULES.name.pattern"
+                    ng-maxlength="$ctrl.FORM_RULES.name.max"
+                    ng-minlength="$ctrl.FORM_RULES.name.min"
+                    placeholder="{{:: 'pci_databases_namespaces_add_name_placeholder' | translate }}"
+                    required
+                />
+            </oui-field>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col">
+            <oui-field>
+                <label class="oui-field__label">
+                    <span
+                        data-translate="{{:: 'pci_databases_namespaces_add_type_label' | translate }}"
+                    ></span>
+                    <small class="text-danger">*</small>
+                </label>
+                <button
+                    type="button"
+                    class="oui-popover-button"
+                    aria-label="Help"
```

Comment:
**issue(blocking):** The popover message is translated, so the label must be translated too (for consistency).

Reply:
Hi Antoine, as seen with team, we don't want the labels to be translated in order for the user to be able to understand how the inputs are linked to the API fields

Reply:
IMHO we should be consistent.

Having non-translated label (here: `Help`) that lead the customer to a popover message that is fully translated sound disturbing.

Checking with the team and we'll keep you posted!

Reply:
Hi Antoine,
Sorry, I think I misunderstood your first comment. The `help` aria-label here is to keep consistency with the `field` component in the ovh-ui-kit: https://github.com/ovh/ovh-ui-kit/blob/master/packages/components/field/src/js/field.html#L13

Reply:
According to the ticket comment, we stick with the help label for now.